src/Nancy/SafePathConfigurationExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Nancy
+{
+    using System.Collections.Generic;
+    using Nancy.Configuration;
+
+    /// <summary>
+    /// Contains <see cref="SafePathConfiguration"/> configuration for <see cref="INancyEnvironment"/>.  
```

Comment:
You're missing "extensions" in this description. Look at the other ones to see what I mean